src/components/Input/NumericInput/NumericInput.tsx

```diff
@@ -116,6 +129,14 @@ export function NumericInput({
       max={max}
       step={step}
       hideControls={hideControls}
+      leftSection={
+        isOutsideRange &&
+        !isEditing && (
+          <ThemeIcon color={'orange.4'} variant={"transparent"}>
+            <MdWarning size={IconSize.sm} />
```

Comment:
Yeah I think making this smaller would be nice
```suggestion
            <MdWarning size={IconSize.xs} />
```